resources/keywords/provision_dbinstance.resource

```diff
@@ -20,15 +20,29 @@ User Navigates To Create Database Instance Screen On Developer View
 
 User Selects DBProvider And Provider Account
     [Arguments]    ${isv_name}=${isv}
+    Set Test Variable    ${isv_name}
     Wait Until Keyword Succeeds    5x    2s    Wait Until Element Is Visible    ${drpDwnDBProvider}
     Wait Until Keyword Succeeds    10s    1s    Toggle Database Provider Drop Down ${isv_name}
     Wait Until Keyword Succeeds     2x      1s      Select Given Provider Account On Add Topology Screen
 
 Toggle Database Provider Drop Down ${isv_name}
+    Run keyword If    "Amazon" in """${isv_name}"""
```

Comment:
Line 29 and 30 can be moved after line 33 to maintain the order of actions like, select Database provider and Then select Engine type